scout/server/blueprints/variant/templates/variant/components.html

```diff
@@ -265,7 +265,7 @@ <h6>No matching causative or managed variants</h6>
             <a href="{{ gene.gnomad_str_link}}" class="btn btn-outline-secondary" target="_blank">GnomAD STR</a>
           {% endif %}
           <a href="{{ gene.panelapp_link }}" class="btn btn-outline-secondary" target="_blank">PanelApp</a>
-          <a href="{{ url_for('variant.marrvel_link', build=case.genome_build, variant_id=variant._id)}}" class="btn btn-outline-secondary" target="_blank">MARRVEL</a>
+          <a href="{{ url_for('variant.marrvel_link', build=case.genome_build or '37', variant_id=variant._id)}}" class="btn btn-outline-secondary" target="_blank">MARRVEL</a>
```

Comment:
It might be more general to have a default generated out of the variant controller, but this works!